src/components/UserDetailSections/PatronBlock/PatronBlock.test.js

```diff
@@ -0,0 +1,64 @@
+import { screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import renderWithRouter from 'helpers/renderWithRouter';
+import PatronBlock from './PatronBlock';
+
+jest.unmock('@folio/stripes/components');
+
+const renderPatronBlock = (props) => renderWithRouter(<PatronBlock {...props} />);
+
+const STRIPES = {
+  config: {},
+  hasPerm: jest.fn().mockReturnValue(true),
+};
+
+const props = {
+  accordionId: 'patronBlocksSection',
+  expanded: true,
+  onToggle: jest.fn(),
+  stripes: STRIPES,
+  match: {
+    params: {
+      id: 'e6dc87a3-591b-43e0-a768-d3552b44878b',
+    }
+  },
+  history: {
+    push: jest.fn(),
+  },
+  intl: { formatMessage: jest.fn() },
+  mutator: {
+    activeRecord: {
+      update: jest.fn(),
+    }
+  },
+  patronBlocks: [{
+    borrowing: true,
+    desc: 'Sample',
+    id: 'f1e0d3e2-fa48-4a82-b371-bea4e44178ab',
+    patronMessage: '',
+    renewals: true,
+    requests: true,
+    staffInformation: '',
+    type: 'Manual',
+    userId: 'e6dc87a3-591b-43e0-a768-d3552b44878b',
+    metadata: { createdDate: '2022-03-01T03:22:48.262+00:00', updatedDate: '2022-03-01T03:22:48.262+00:00' }
+  }]
+};
+
+describe('PatronBlock component', () => {
+  beforeEach(() => {
+    renderPatronBlock(props);
+  });
+  it('it must be rendered', () => {
+    expect(screen.getByText('ui-users.settings.patronBlocks')).toBeInTheDocument();
+  });
+  it('Checking if it navigates to patronBlock', () => {
+    userEvent.click(document.querySelector('[data-row-inner="0"]'));
+    expect(screen.getByText('Manual')).toBeInTheDocument();
+  });
+  it('checking for sort order', () => {
+    userEvent.click(document.querySelector('[id="clickable-list-column-blockedactions"]'));
+    expect(screen.getByText('Sample')).toBeInTheDocument();
```

Comment:
Same as above. Additionally, do you think it's expected that changing the sort (I think that's what you mean by 'checking for sort order') would change the content? I would expect the content to remain the same, but the order to change, and thus we would need to check for position in the document, not just presence in it.

Reply:
Created a issue UIU-2358 for this and added in the test file as comment